Website/src/rendering/src/components/HeroSection.tsx

```diff
@@ -1,50 +1,39 @@
+import { Text, Field, RichText, ImageField, Placeholder } from '@sitecore-jss/sitecore-jss-nextjs';
 import { ComponentProps } from 'lib/component-props';
-import { Text, Field, LinkField, Link, ImageField, Image } from '@sitecore-jss/sitecore-jss-nextjs';
 
 export type HeroProps = ComponentProps & {
   fields: {
     Hero: ImageField;
-    Logo: ImageField;
     Slogan: Field<string>;
-    Expo: Field<string>;
+    Eyebrow: Field<string>;
     Title: Field<string>;
-    Subtitle: Field<string>;
-    When: Field<string>;
-    Link: LinkField;
+    Body: Field<string>;
   };
 };
 
-const HeroSection = ({ fields }: HeroProps): JSX.Element => {
+const HeroSection = (props: HeroProps): JSX.Element => {
   return (
     <section
       className="section__hero banner"
-      style={{ backgroundImage: `url("${fields.Hero?.value?.src}")` }}
+      style={{ backgroundImage: `url("${props.fields.Hero?.value?.src}")` }}
```

Comment:
I am not able to test the PR on a portal instance as the image does not load because of the CH port issue. Should we wait for this issue to be resolved before we merge this PR?

Reply:
I was able to add the port by editing the field raw value. It should work once the port issue is fixed.